api/sixtyfps-rs/sixtyfps-macros/lib.rs

```diff
@@ -357,10 +357,10 @@ pub fn sixtyfps(stream: TokenStream) -> TokenStream {
         .filter(|path| path.is_absolute() && !path.ends_with("Cargo.toml"))
         .filter_map(|p| p.to_str())
         .map(|p| quote! {const _ : &'static [u8] = ::core::include_bytes!(#p);});
-    result.as_mut().map(|x| {
+    if let Some(x) = result.as_mut() {
```

Comment:
Irrelevant comment: This is strange that `map()` returning void is considered non idiomatic, while  `?` returning void is.